dll/win32/windowscodecs/CMakeLists.txt

```diff
@@ -79,6 +70,12 @@ add_library(windowscodecs MODULE
     version.rc
     ${CMAKE_CURRENT_BINARY_DIR}/windowscodecs.def)
 
+if(MSVC)
+    target_compile_options(windowscodecs PRIVATE
+        /wd4133 # error C4133: 'function': incompatible types - from 'WICPixelFormatNumericRepresentation *' to 'DWORD *'
```

Comment:
What does this change wrt. "/we4133" from before? (in other cmakelists files you instead used the remove_target_compile_option function for such things)

Reply:
It disables the warning, instead of treating it as error, whereas removing /we keeps it but doesn't treat it as an error.

The use of this is inconsistent through the whole code tree. At some point we will have to agree about what should be done...